sudoexec/jstop

```diff
@@ -213,7 +213,8 @@ sleep 2
 external_exec_script -s stop.d
 if [ ${vnet} -eq 1 ]; then
 	if [ "${ip4_addr}" = "REALDHCP" ]; then
-		${JEXEC_CMD} ${jname} /bin/pkill -9 dhclient
+		${JEXEC_CMD} ${jname} /bin/pkill -9 dhclient 2>/dev/null 1>&2
```

Comment:
As far as I can see, we can do one pkill call ( pkill pattern ... ), e.g:

${JEXEC_CMD} ${jname} /bin/pkill -9 dhclient dhcpd 2>/dev/null 2>&1